qfdmo/models/acteur.py

```diff
@@ -420,7 +421,29 @@ def numero_et_complement_de_rue(self) -> str | None:
         return result.replace("bis", "b")
 
     @property
-    def nom_sans_adresse_et_complement(self) -> str | None:
+    def code_departement(self) -> str | None:
+        """Extrait le code départemental du code postal"""
+        cp = self.code_postal
+        if not cp:
+            return None
+        cp = str(cp).strip()
+        return None if not cp or len(cp) != 5 else cp[:2]
+
+    @property
+    def combine_adresses(self) -> str | None:
+        """Combine les valeurs de tous les champs adresses,
+        util notamment pour la clustering car les sources mélangent
+        un peut tout à travers ces champs et on veut tout regrouper
+        pour augmenter les chances de similarité (sachant qu'on
+        peut itiliser à posteriori fonctions de normalisations qui suppriment
+        les mots en doublons et augmentent encore la similarité)"""
+        result = (
+            (self.adresse or "").strip() + " " + (self.adresse_complement or "").strip()
```

Comment:
```suggestion
            f"{(self.adresse or "").strip()} {(self.adresse_complement or "").strip()"
```

on peut peut-être gagner en lisibilité en utilisant une f-string